manifest.php

```diff
@@ -30,7 +30,7 @@
     'label' => 'extension-tao-dac-simple',
     'description' => 'extension that allows admin to give access to some resources to other people',
     'license' => 'GPL-2.0',
-    'version' => '6.5.1',
+    'version' => '6.6.0',
```

Comment:
Shall we add dependency on taskQueue here?

Reply:
Added